myapp/spec/feature/tasks_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.feature Task, type: :feature, js: true do
+  # task-C
+  feature '新規登録画面' do
+    background do
+      # タスク新規登録画面へ遷移
+      visit new_task_path
+    end
+    context 'フォームの入力値が正常の場合' do
+      scenario 'タスクの新規作成が成功' do
+        input_name = 'ガス閉栓手続き'
+        input_description = '京葉ガスに連絡・日付確定'
+
+        # フィールドに入力
+        fill_in 'タスク名', with: input_name
+        fill_in '詳細', with: input_description
```

Comment:
ここのラベル名？は他のテストでも使ってるのでletとかで変数化した方が楽かもしれません！
あとで名前変わった時とか
```
        fill_in 'タスク名', with: input_name
        fill_in '詳細', with: input_description
```

```suggestion
RSpec.feature Task, type: :feature, js: true do
  # task-C
  feature '新規登録画面' do

    let(:task_form_name) { 'タスク名' }
    let(:detail_form_name) { '詳細' }
    let(:register_button_name) { '登録' }

    background do
      # タスク新規登録画面へ遷移
      visit new_task_path
    end
    context 'フォームの入力値が正常の場合' do
      scenario 'タスクの新規作成が成功' do
        input_name = 'ガス閉栓手続き'
        input_description = '京葉ガスに連絡・日付確定'

        # フィールドに入力
        fill_in task_form_name, with: input_name
        fill_in detail_form_name with: input_description
```

Reply:
ラベルを変数化しました！
（自分で書いて・動かしてとやりたかったので、Suggested change は参考にさせていただくにとどめました。）

Reply:
Suggestedは割と雑に書いたのでご自身で書いていただいてよかったですw